app/helpers/application_helper.rb

```diff
@@ -53,7 +53,7 @@ def date_time_relative_value(time)
   end
 
   def iana_timezone
-    ActiveSupport::TimeZone::MAPPING[Time.zone.try(:name)] || 'UTC'
+    ActiveSupport::TimeZone::MAPPING.has_value?(Time.zone.try(:name)) ? Time.zone.try(:name) : ActiveSupport::TimeZone::MAPPING[Time.zone.try(:name)] || 'UTC'
```

Comment:
I have been investigating this bit more and I believe the correct zone identifier is in the wrapped instance `.tzinfo` so I believe we can even simplify this to:
```suggestion
    Time.zone&.tzinfo&.name || 'UTC'
```
Note I am using `&.` operator, but we can substitute for `.try(:tzinfo).try(:name)`

Reply:
`try` is Rails and `&` is Ruby.  Personally I love `&`

Reply:
@jeremylenz I agree, the only fallback of the operator is that it's still bit young, so it's breaking compatibility with `ruby < 2.3.0` but in foreman we don't mind :)